visual-studio-devenv.cmd

```diff
@@ -0,0 +1,47 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+setlocal enableextensions
+setlocal enabledelayedexpansion
+
+if defined VisualStudioVersion goto :Run
+
+set _VSWHERE="%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe"
+if exist %_VSWHERE% (
+  for /f "usebackq tokens=*" %%i in (`%_VSWHERE% -latest -prerelease -property installationPath`) do set _VSCOMNTOOLS=%%i\Common7\Tools
+)
+if not exist "%_VSCOMNTOOLS%" set _VSCOMNTOOLS=%VS140COMNTOOLS%
+if not exist "%_VSCOMNTOOLS%" (
+    echo Error: Visual Studio 2015 or 2017 required.
+    echo        Please see https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md for build instructions.
```

Comment:
Should point to `core-setup`'s developer/building guide, not `corefx`.

Reply:
I copied this from core-setup rum.cmd.  core-setup build documentation is mostly absent at the moment.  As we resolve this I agree it should be fixed.  I'll take a look at the windows build instructions.  Perhaps they are already adequate.

Reply:
https://github.com/dotnet/core-setup/blob/master/Documentation/building/windows-instructions.md